gamestonk_terminal/portfolio_analysis/portfolios/my_portfolio.csv

```diff
@@ -1,6 +1,6 @@
-﻿Ticker,Shares,Other
-AAPL,5,
-GME,5,lol
-MSFT,3,
-O,2,s
+﻿Ticker,Shares,Other
+AAPL,5,
+GME,5,lol
```

Comment:
lol

Reply:
I've added 420 shares of AMC, and 69 of GME eheh